docker/dockerfile

```diff
@@ -0,0 +1,63 @@
+#
+# Self-contained KWIVER installation
+# 
+
+FROM ubuntu:trusty
+
+#
+# System dependencies
+#
+
+RUN apt-get update -y && apt-get install -y && sudo apt-get install curl -y \ 
+&& apt-get install build-essential libgl1-mesa-dev -y \
+&& apt-get install libexpat1-dev -y \
+&& apt-get install libgtk2.0-dev -y \
+&& apt-get install libxt-dev -y \
+&& apt-get install libxml2-dev -y \
+&& apt-get install libssl-dev -y \
+&& apt-get install openssl libssl-dev -y \
+&& apt-get install liblapack-dev -y \
+&& apt-get install zlib1g-dev -y \
+&& apt-get install python2.7-dev -y \
+&& apt-get install python-pip -y \
+&& pip install numpy \
+&& apt-get install git -y
+
+#
+# CMake
+#
+
+ENV PATH $PATH:/cmake/cmake-3.11.3/bin  
+RUN mkdir /cmake \
+&& cd /cmake \
+&& curl -O "https://cmake.org/files/v3.11/cmake-3.11.3.tar.gz" \
+&& tar -xvf cmake-3.11.3.tar.gz \
+&& cd cmake-3.11.3 \
+&& ./configure \
+&& make -j8 -k \
+
+#
+# Building Fletch
+#
+
+&& mkdir /fletch /fletch/build /fletch/build/rel /fletch/build/deb \
+&& cd /fletch \
+&& git clone https://github.com/Kitware/fletch.git src \
+&& cd src \
+&& git checkout release \ 
+&& cd /fletch/build/rel \ 
+&& cmake -DCMAKE_BUILD_TYPE=Release -Dfletch_ENABLE_ALL_PACKAGES=ON -Dfletch_ENABLE_PYTHON=ON ../../src \
+&& make -j8 -k \
+
+#
+# Building Kwiver
+#
+
+&& mkdir /kwiver /kwiver/src /kwiver/build /kwiver/build/release /kwiver/build/debug \
```

Comment:
Also looks like we don't use /kwiver/build/debug

Reply:
There might be some debate, but I contend that we don't really need to use the fletch build source or build products anymore, once this is done. You should probably set the fletch_BUILD_INSTALL_PREFIX to a directory which will persist and rm -rf /fletch/build/rel  /fletch/src so the image size doesn't explode.